scripts/install-latest-go.sh

```diff
@@ -1,10 +1,20 @@
 #!/usr/bin/env bash
 
-LATEST_GO_VERSION=$(curl https://go.dev/VERSION?m=text)
-INSTALLED_GO_VERSION=$(go version | grep -oP "go\d+\.\d+\.\d+")
+REQUIRED_GO_VERSION="1.20.2"
+INSTALLED_GO_VERSION="$(go version | { read -r _ _ v _; echo "${v#go}"; })"
 
-if [ "$INSTALLED_GO_VERSION" != "$LATEST_GO_VERSION" ]; then {
-    wget https://go.dev/dl/"${LATEST_GO_VERSION}".linux-amd64.tar.gz
-    rm -rf /usr/local/go && tar -C /usr/local -xzf "${LATEST_GO_VERSION}".linux-amd64.tar.gz
-    rm "${LATEST_GO_VERSION}".linux-amd64.tar.gz
-} fi
+function version { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+
+if [ "$(version "${INSTALLED_GO_VERSION}")" -ge "$(version ${REQUIRED_GO_VERSION})" ]; then
+    echo "Version is up to date"
+else
+    unameOut="$(uname -s)"
+    case "${unameOut}" in
+        Linux*)
+                    wget https://go.dev/dl/"${REQUIRED_GO_VERSION}".linux-amd64.tar.gz
+                    rm -rf /usr/local/go && tar -C /usr/local -xzf "${REQUIRED_GO_VERSION}".linux-amd64.tar.gz
+                    rm "${REQUIRED_GO_VERSION}".linux-amd64.tar.gz
+                    ;;
+        *)          echo "Install go with version ${REQUIRED_GO_VERSION}"
```

Comment:
```suggestion
        *)          echo "Installed go with version ${REQUIRED_GO_VERSION}"
```

Reply:
That is an imperative sentence. It is up to the user on non-linux platform to install go based on his platform.